doc/backend/norms-everkuendung-api-v1.yaml

```diff
@@ -270,11 +273,80 @@ components:
           type: string
           format: uri
           description: Eine URI, die die Art des Fehlers ausweist.
-          example: /errors/example-error
+          example: /errors/norm-with-eli-exists-already
         title:
           type: string
           description: Eine kurze, menschenlesbare Beschreibung des Problems. Diese Information ist zum Debugging gedacht und sollte nicht in einer Benutzeroberfläche verwendet werden.
           example: Ein Beispielfehler ist aufgetreten.
         detail:
           type: string
           description: Menschenlesbare Details zur Ursache des Problems. Diese Information ist zum Debugging gedacht und sollte nicht in einer Benutzeroberfläche verwendet werden.
+        errors:
+          type: array
+          description: Eine Liste strukturierter Validierungsfehler aus XSD oder Schematron.
+          items:
+            oneOf:
+              - $ref: "#/components/schemas/XsdValidationError"
+              - $ref: "#/components/schemas/SchematronValidationError"
+        extensions:
```

Comment:
Do we need these extensions? I thought we wanted to have a "fix" structure of what can be returned by us and we just wanted to have the type, title and details right?